src/ontology/mondo-edit.obo

```diff
@@ -619303,6 +619303,17 @@ relationship: excluded_from_qc_check http://purl.obolibrary.org/obo/mondo/sparql
 relationship: has_material_basis_in_germline_mutation_in http://identifiers.org/hgnc/4585 {source="PMID:27683935"} ! GRIN2A
 property_value: IAO:0000233 "https://github.com/monarch-initiative/mondo/issues/9063" xsd:anyURI
 
+[Term]
+id: MONDO:7770001
+name: equine juvenile spinocerebellar ataxia, FDXR-related, horse
+def: "Any spinocerebellar ataxia that occurs in horses due to a variation in the FDXR gene. EJSCA is a degenerative axonopathy characterized by acute onset of ataxia within the first month of life, affecting the pelvic limbs more severely than the thoracic limb and rapidly progressing to inability to stand." [doi:10.1016/j.jevs.2025.105561, PMID:38669583]
+synonym: "EJSCA" EXACT ABBREVIATION [PMID:38669583]
+is_a: MONDO:1012960 {source="PMID:38669583"} ! hereditary cerebellar ataxia, non-human animal
+relationship: has_material_basis_in_germline_mutation_in http://identifiers.org/ncbigene/100060312 {source="PMID:38669583"}
+relationship: in_taxon NCBITaxon:9796 {source="PMID:38669583"} ! Equus caballus
```

Comment:
@katiermullen based on the textual definition, it looks like you can add a logical definition to, including the taxon and the gene. 
Would you agree? Or are there other disease that would fit the definition?